src/server_manager/web_app/app.spec.ts

```diff
@@ -35,41 +32,43 @@ const TOKEN_WITH_ONE_SERVER = 'one-server-token';
 // tslint:disable-next-line:no-any
 (global as any).bringToFront = () => {};
 
+// Inject app-root element into DOM once before the test suite runs
+beforeAll(async () => {
+  const loadAppRoot = new AppRoot();  // TODO: Remove once figure out how to load AppRoot properly.
+
+  const appRootEl = document.createElement('app-root');
```

Comment:
Why do you need to create this element? You already created an AppRoot above.

Reply:
I tried to create the AppRoot instance directly and add it to the DOM using `appendChild` but I ran into some issues.

I suspect they were double casting issues similar to the casting we do to fetch app-root:
`document.getElementById('appRoot') as unknown as AppRoot`

I will look into this further.